docs/release-notes/breaking-changes.md

```diff
@@ -195,6 +195,81 @@ We’ve removed deprecated SSL settings in some {{ls}} plugins, and have replace
+
+
+### Deprecated configuration settings removed [removed-params]
+
+We have removed support for previously deprecated configuration settings:
+
+#### `http.*` prefixed settings for the {{ls}} API
+When configuring the {{ls}} API, settings prefixed by `http.*` have been replaced by the equivalent settings prefixed with `api.*`. [#16552](https://github.com/elastic/logstash/pull/16552)
+
+#### `event_api.tags.illegal`
+
+Any events that include field named tags automatically rename the field _tags to avoid any clash
+with the reserved {{ls}} tags field. 
+Instead, {{ls}} generates `_tagsparsefailure` in the event `tags` and the illegal value is written to the `_tags` field. [#16461](https://github.com/elastic/logstash/pull/16461)
+
+
+### Ingest converter removed [removed-ingest-converter]
+
+The ingest converter, which was previously used to convert ingest pipelines to {{ls}} pipelines, has been removed.
+
+For more information, check [#16453](https://github.com/elastic/logstash/pull/16453)
+
+
+### JDK11 not supported [jdk-11-support-drop]
+
+JDK17 is the minimum version of the JDK required to run Logstash.
+For the best experience, we still recommend running {{ls}} using the bundled-jdk. 
+See [Logstash JVM requirements](/reference/getting-started-with-logstash.md#ls-jvm)
+for details. [#16443](https://github.com/elastic/logstash/pull/16443)
+
+### Docker base image now UBI9 based [docker-base-image-change]
+
+{{ls}} on Docker, the base image has been changed from Ubuntu to UBI9. If you create a Docker image based on the
+{{ls}} image, and rely on it being Ubuntu based, you will need to change your derived image to take account of this
+change.
+
+For more information, check [#16599](https://github.com/elastic/logstash/pull/16599)
+
+### Cannot run {{ls}} as `superuser` by default [disallow-superuser]
+
+We've changed the default behavior to prevent users from accidentally running {{ls}} as a superuser.
+If you try to run {{ls}} as a superuser, it logs an error and fails to start, ensuring that users cannot run Logstash with elevated privileges by accident.
+
+You can change the value of the `allow_superuser` setting to `true` in [logstash.yml](/reference/logstash-settings-file.md) if you want to restore the previous behavior and allow {{ls}} to run with superuser privileges. [#16558(https://github.com/elastic/logstash/pull/16558)
+
+### New setting required to continue using legacy internal monitoring [allow-legacy-monitoring]
+
+To continue using deprecated internal collection to monitor {{ls}}, set `xpack.monitoring.allow_legacy_collection` to `true` in [logstash.yml](/reference/logstash-settings-file.md).
+We encourage you to move to [agent-driven monitoring](reference/monitoring-logstash-with-elastic-agent.md), the latest, supported way to monitor Logstash [#16586(https://github.com/elastic/logstash/pull/16586)
```

Comment:
```suggestion
We encourage you to move to [agent-driven monitoring](reference/monitoring-logstash-with-elastic-agent.md), the latest, supported way to monitor Logstash [#16586](https://github.com/elastic/logstash/pull/16586)
```

Reply:
Two possible fixes in this line:
- Fix the path to elastic agent topic
- Add missing `]` to fix link.